contracts/bridge/Bridge.sol

```diff
@@ -105,69 +110,72 @@ contract Bridge is IERC20BridgeReceiver, IERC721BridgeReceiver, Ownable, BridgeF
 
     // handleERC20Transfer sends the token by the request.
     function handleERC20Transfer(
-        uint256 _amount,
+        address _from,
         address _to,
-        address _contractAddress,
+        address _tokenAddress,
+        uint256 _value,
         uint64 _requestNonce,
         uint64 _requestBlockNumber
```

Comment:
Can I ask what '_requestBlockNumber' or lastHandledRequestBlockNumber is used for?

Reply:
`_requestBlockNumber ` is the block (including request value transfer tx)'s number. :)

`lastHandledRequestBlockNumber` is the nonce of handled request event's nonce.

Reply:
I will explain offline if you need. :)

Reply:
Thank you for explaining!! :)